src/controllers/transactions.controller.ts

```diff
@@ -43,7 +43,7 @@ export class TransactionsController {
   ): Promise<Response<{ data: Transaction[] }>> {
```

Comment:
The name of this method, `createMultiTransaction`, seems somewhat misleading. Can we maybe adopt a method naming scheme from `TransactionService` for this and the other methods in this class (`getOneById`, `addMany`, `updateOne`, etc.)?